src/coreComponents/unitTests/wavePropagationTests/testWavePropagationAcousticFirstOrder.cpp

```diff
@@ -116,14 +116,14 @@ char const * xmlInput =
   "    <FieldSpecification\n"
   "      name=\"cellVelocity\"\n"
   "      initialCondition=\"1\"\n"
-  "      objectPath=\"mesh/FE1/ElementRegions/Region/cb\"\n"
+  "      objectPath=\"ElementRegions/Region/cb\"\n"
```

Comment:
@acitrain I changed the `objectPath` of your unit test because the `FieldSpecification` was silently ignored

Reply:
> @acitrain I changed the `objectPath` of your unit test because the `FieldSpecification` was silently ignored

Ok no problem @francoishamon. By this may be the reason why this test does not pass in @labesse40 last PR (that's weird but why not....) 
Thank you for noticing this !